pep-0698.rst

```diff
@@ -308,11 +308,60 @@ use it will type-check as before, without the additional type safety.
 Runtime Behavior
 ================
 
-At runtime, ``@typing.override`` will do nothing but return its argument.
+Set ``__override__ = True`` when possible
+-----------------------------------------
+
+At runtime, ``@typing.override`` will make a best-effort attempt to add an
+attribute ``__override__`` with value ``True`` to its argument. By "best-effort"
+we mean that we will try adding the attribute, but if that fails (for example
+because the input is a descriptor type with fixed slots) we will silently
+return the argument as-is.
+
+This is exactly what the ``@typing.final`` decorator does, and the motivation
+is similar - it gives runtime libraries the ability to use ``@override``. As a
+concrete example, a runtime library could check ``__override__`` in order
+to automatically populate the ``__doc__`` attribute of child class methods
+using the parent method docstring.
+
+Limitations of setting ``__override__``
+---------------------------------------
+
+As described above, adding ``__override__`` may fail at runtime, in which
+case we will simply return the argument as-is.
+
+In addition, even in cases where it does work it may be difficult for users
+to correctly work with multiple decorators, because getting the ``__override__``
+field to exist on the final output requires understanding the implementation
+of each decorator:
+
+- The ``@override`` decorator needs to execute *after* ordinary decorators
+  like ``@functools.lru_cache`` that use wrapper functions, since we want to
+  set ``__override__`` on the outermost wrapper. This means it needs to
+  go *above* all these other decorators.
+- But ``@override`` needs to execute *before* many special desriptor-based
+  decorators like ``@property``, ``@staticmethod``, and ``@classmethod``.
+- As discussed above, in some cases (for example a descriptor with fixed
+  slots or a descriptor that also wraps) it may be impossible to get the
+  ``__override__`` attribute at all.
+
+As a result, runtime support for setting ``__override__`` is best effort
+only, and we do not expect type checkers to validate the ordering of
+decorators.
 
-We considered other options but rejected them because the downsides seemed to
-outweigh the benefits, see the Rejected Alternatives section.
+Lack of any clear benefit
```

Comment:
Should this be removed?

Reply:
Oh yes, this was a copy-paste error (I was moving the discussion of limitations up from the old "Rejected Alternatives section"